src/mon/Monitor.cc

```diff
@@ -464,6 +464,7 @@ const char** Monitor::get_tracked_conf_keys() const
     "clog_to_graylog",
     "clog_to_graylog_host",
     "clog_to_graylog_port",
+    "host",
```

Comment:
These tracked keys work as filters, so we can react properly upon "host" configuration changing.
```c++
void Monitor::handle_conf_change(const struct md_config_t *conf,
                                 const std::set<std::string> &changed)
...
 if (changed.count("clog_to_monitors") ||
            changed.count("clog_to_syslog") ||
            changed.count("clog_to_syslog_level") ||
            changed.count("clog_to_syslog_facility") ||
            changed.count("clog_to_graylog") ||
            changed.count("clog_to_graylog_host") ||
            changed.count("clog_to_graylog_port") ||
            changed.count("host") ||
            changed.count("fsid"))
    {
        update_log_clients();
    }
```
E.g., if we change "host" only, the above logic won't catch it because "host" is not tracked.